### ISSUE TITLE
keys function can't return Infinity ttl of key

### ISSUE THREAD

**Issue by WongYT** [isaacs/ttlcache#27]

**Comment by Julien-R44** [comment]
I believe this is definitely a bug. In my case, a workaround was to use `cache.data`. although it's not typed, it allows you to iterate through your keys, including those that never expire

**Comment by isaacs** [comment]
Yes, it is a bug. Patch welcome, or wait until I get to it. Busy trying to finish tap, and o have a few node core and ts node things to spend some attention on, but ttlcache is on the list for sure.

**Comment by woifes** [comment]
Replace `Infinity` with `Number.MAX_SAFE_INTEGER` as a workaround until the PR is merged.
